libs/portals/my-pages/core/src/components/SortableTable/useSortableData.ts

```diff
@@ -14,12 +44,34 @@ export const useSortableData = <T extends Record<string, unknown>>(
       const keyA = a[sortConfig.key as keyof T]
       const keyB = b[sortConfig.key as keyof T]
 
-      if (!(typeof keyA === 'string') || !(typeof keyB === 'string')) {
+      if (keyA === undefined || keyB === undefined) {
         return 0
       }
 
       const multiplier = sortConfig.direction === 'ascending' ? 1 : -1
-      return keyA.localeCompare(keyB, undefined, { numeric: true }) * multiplier
+
+      if (
+        isValidDateString(keyA as string) &&
+        isValidDateString(keyB as string)
+      ) {
+        const dayA = decunstructDateString(keyA as string)
+        const dayB = decunstructDateString(keyB as string)
+        if (!dayA || !dayB) return 0
+        const dateA = new Date(dayA.year, dayA.month - 1, dayA.day)
+        const dateB = new Date(dayB.year, dayB.month - 1, dayB.day)
+
+        return (dateA.getTime() - dateB.getTime()) * multiplier
+      } else if (typeof keyA === 'string' && typeof keyB === 'string') {
+        return (
+          keyA.localeCompare(keyB, undefined, { numeric: true }) * multiplier
```

Comment:
Why use `localeCompare` if the locale is undefined? What is the default locale in that case?

Reply:
> "If you omit this parameter, the default locale of the JavaScript runtime is used."